src/schema/nodes/manuscript.ts

```diff
@@ -29,7 +29,7 @@ export interface ActualManuscriptNode extends ManuscriptNode {
 
 export const manuscript: NodeSpec = {
   content:
-    'title? contributors? affiliations? keywords? supplements? abstracts body backmatter comments',
+    'title? contributors? affiliations? keywords? supplements? abstracts? body? backmatter? comments?',
```

Comment:
I don't think there is value in making comments optional. Can we also keep abstracts/body/backmatter required? I also feel title should be required but jats import might break with this.

Reply:
reverted